### PR TITLE
ENH: add multilinux2010

### PR DESCRIPTION
**Pull request by mattip**
Now that multilinux2010 has been released, update the demo to use it

~And as an extra, replace nose with pytest~

Edit: do not change from nose

### CODE REVIEW

**Comment by mattip** [comment]
xref pypa/manylinux#179

**Comment by takluyver** [comment]
I think you opened this before the manylinux2010 image had finished building; that should clear up next time the tests run.

You also seem to have a problem running pytest - I think possibly you need the `--pyargs` option. Or leave it as nose for now and work on that in another PR.

**Comment by mattip** [comment]
removed the nose -> pytest changes

**Comment by takluyver** [comment]
This now passes, but the wheels built on the manylinux2010 image appear to get a `manylinux1` tag instead of `manylinux2010`. We'll need to investigate whether we're doing something wrong here, or in the build image, or in auditwheel.

File: .travis.yml [hunk 1]
```diff
@@ -7,15 +7,22 @@ matrix:
...
+        - docker
+      env: DOCKER_IMAGE=quay.io/pypa/manylinux2010_x86_64
+           PLAT=manylinux1
 
 install:
   - docker pull $DOCKER_IMAGE
 
 script:
-  - docker run --rm -v `pwd`:/io $DOCKER_IMAGE $PRE_CMD /io/travis/build-wheels.sh
+  - docker run --rm -v `pwd`:/io $DOCKER_IMAGE $PRE_CMD -e PLAT=$PLAT /io/travis/build-wheels.sh
```
Comment (takluyver) [inline .travis.yml]: I think this would have to go before `$PRE_CMD`.

File: .travis.yml [hunk 2]
```diff
@@ -7,15 +7,22 @@ matrix:
...
       services:
         - docker
       env: DOCKER_IMAGE=quay.io/pypa/manylinux1_i686
            PRE_CMD=linux32
+           PLAT=manylinux1
+    - sudo: required
+      services:
+        - docker
+      env: DOCKER_IMAGE=quay.io/pypa/manylinux2010_x86_64
+           PLAT=manylinux1
```
Comment (takluyver) [inline .travis.yml]: 2010?

Also, the options for auditwheel need the full platform tag, like `manylinux2010_x86_64`.

File: .travis.yml (hunk 1 above)
Comment (mattip) [reply .travis.yml]: right. Sorry, for the noise, I should find a way to test locally before pushing

File: .travis.yml (hunk 2 above)
Comment (mattip) [reply .travis.yml]: fixing. What is the 32 bit manylinux1_* full tag?

Comment (mattip) [reply .travis.yml]: trying the docker image name

**Comment by takluyver** [comment]
That looks good to me. :-)

We could get rid of the DOCKER_IMAGE variable by using `quay.io/pypa/$PLAT` - I don't know if it's clearer to be more concise, or to spell out the image names separately. :-)

**Comment by mattip** [comment]
Since this is a demo, I would prefer to be as explicit as possible.

Interesting that auditwheel says "INFO:auditwheel.main_repair:Wheel is eligible for a higher priority tag. You requested manylinux2010_x86_64 but I have found this wheel is eligible for manylinux1_x86_64." and then goes and make both wheels, manylinux1 and manylinux2010

**Comment by takluyver** [comment]
OK, explicit is fine by me.

Yes, I think I would have made auditwheel show the message but only add the requested tag, even if another one is valid. What it's doing is reasonable enough, though.

**Comment by mattip** [comment]
anything else I need to do?

**Comment by takluyver** [comment]
This looks good to me, and I've now been granted the power to merge it :slightly_smiling_face:

**Pull request merged**